packages/create-bison-app/template/package.json.ejs

```diff
@@ -44,42 +40,33 @@
     "test:server": "next start --port 3001",
     "ts-node": "ts-node-dev --project tsconfig.cjs.json -r tsconfig-paths/register",
     "withEnv:test": "dotenv -c test --",
-    "watch:all": "concurrently -n \"NEXUS,GQLCODEGEN,TYPESCRIPT\" -c \"black.bgGreen.dim,black.bgBlue.dim,white.bgMagenta.dim\" \"yarn watch:nexus\" \"yarn watch:codegen\" \"yarn watch:ts\"",
-    "watch:codegen": "yarn codegen --watch",
-    "watch:nexus": "yarn ts-node --transpile-only --respawn --watch graphql/schema.ts,prisma/schema.prisma graphql/schema.ts",
     "watch:ts": "yarn dev:typecheck --watch"
   },
   "dependencies": {
-    "@apollo/client": "^3.6.1",
     "@chakra-ui/react": "^2.2.0",
     "@chakra-ui/theme": "^2.1.0",
     "@emotion/react": "^11.0.0",
     "@emotion/styled": "^11.0.0",
     "@prisma/client": "^4.0.0",
-    "apollo-server-micro": "^3.6.7",
+    "@tanstack/react-query": "^4.8.0",
```

Comment:
We probably should bump this to the latest version, or even wait to merge this until tRPC 10 is out of beta. Though in my testing it's been very stable.